### ISSUE TITLE
Support Team Names in webhooks

### ISSUE THREAD

**Issue by mefellows** [pactflow/roadmap#42]

**Comment by MajorCaiger** [comment]
Hi @mefellows 
I can add a bit more context here.

We're investigating the use of Webhooks to gather metrics about which teams are actively publishing/verifying pacts.

I was thinking maybe 2 new variables, one for consumerTeams and one for providerTeams which just contain a comma-separate list of teams associates with those pacticipants.

**Comment by mefellows** [comment]
Thanks @MajorCaiger. Is the idea that you're using webhooks to fire to a metrics endpoint or something?

Is there an alternative approach where we could be exposing more metrics e.g. through the API (e.g. adding more to the `/metrics` endpoin), for an example?

**Comment by MajorCaiger** [comment]
Yeah that might work @mefellows.  Could we possibly have a filter, so you can view an individual team's metrics?
